### PR TITLE
publish snapshots

### PR DESCRIPTION
**Pull request by bavardage**

### CODE REVIEW

File: gradle/publish.gradle [hunk 1]
```diff
@@ -21,12 +25,29 @@ publishing {
             artifact(sourceJar) {
                 classifier 'sources'
```
Comment (markelliot) [inline gradle/publish.gradle]: can drop this too

File: gradle/publish.gradle [hunk 2]
```diff
@@ -14,6 +15,9 @@ task javadocJar(type: Jar, dependsOn: javadoc) {
 
 tasks.build.dependsOn sourceJar, javadocJar
 
+
+// hack due to limitations of
```
Comment (markelliot) [inline gradle/publish.gradle]: not necessary?

File: gradle/publish.gradle [hunk 3]
```diff
@@ -21,12 +25,29 @@ publishing {
...
+            password = System.env.BINTRAY_KEY
+            maven = true
+        }
+        defaults {
+            publications (publishing.publications.bintray)
+            properties = ['git': project.version] 
         }
     }
 }
+artifactoryPublish.dependsOn 'generatePomFileForJfrogPublication', 'sourceJar', 'build'
```
Comment (markelliot) [inline gradle/publish.gradle]: don't need to list sourceJar here since 'build' depends on it above

File: gradle/publish.gradle (hunk 1 above)
Comment (bavardage) [inline gradle/publish.gradle]: :white_check_mark:

File: gradle/publish.gradle (hunk 2 above)
Comment (bavardage) [inline gradle/publish.gradle]: oops, done :white_check_mark:

File: gradle/publish.gradle (hunk 3 above)
Comment (bavardage) [inline gradle/publish.gradle]: :white_check_mark:

File: gradle/publish.gradle [hunk 4]
```diff
@@ -18,16 +19,31 @@ publishing {
...
+            password = System.env.BINTRAY_KEY
+            maven = true
+        }
+        defaults {
+            publications (publishing.publications.bintray)
+            properties = ['git': project.version]
+        }
+    }
+}
+artifactoryPublish.dependsOn 'generatePomFileForJfrogPublication', 'build'
```
Comment (markelliot) [inline gradle/publish.gradle]: `generatePomFileForBintrayPublication` ... it's actually some string patterning that the publish plugin uses to create the task names

**Comment by bavardage** [comment]
verified this works locally
https://oss.jfrog.org/artifactory/webapp/#/builds/encrypted-config-value/1452030651808/1452030651242/general/

**Comment by markelliot** [comment]
ergh, just after I merged this I see that you only published encrypted-config-value and not also encrypted-config-value-bundle?

**Comment by bavardage** [comment]
hmm, my local test published them I think
https://oss.jfrog.org/artifactory/webapp/#/builds/encrypted-config-value-bundle/1452030651808/1452030651242/general/

will investigate

**Pull request merged**